discord-bot/.env.example

```diff
@@ -1,6 +1,10 @@
 DISCORD_GUILD_ID=test
 DISCORD_CLIENT_ID=test
 DISCORD_TOKEN=test
+#'ASTRA_JSON_API_URL' takes precedence over 'STARGATE_JSON_API_URL' when both are present, and the token is expected to be present as part of the 'ASTRA_JSON_API_URL'
+#'ASTRA_JSON_API_URL' format : https://${ASTRA_DB_ID}-${ASTRA_DB_REGION}.apps.astra.datastax.com/api/json/v1/${ASTRA_KEYSPACE}?applicationToken=${ASTRA_DB_APPLICATION_TOKEN}
```

Comment:
I think this would be more usable if we had separate environment variables for $ASTRA_DB_ID, $ASTRA_DB_REGION, $ASTRA_KEYSPACE and $ASTRA_DB_APPLICATION TOKEN, and compose the URL from that.

Reply:
We could also hardcode `astra-test` in the URL in the short term since the API is not available on Astra yet. Or expose it as an option. I don't have a strong opinion on this point.

Reply:
I thought about it too. For a couple of reasons, I thought of doing it this way. 
1. Users can simply copy the URL from swagger-UI or the the URL under the `Connect` tab on the Astra UI and just append the token to the URL.
2. For 'test'/'dev' environments, the url becomes .apps.astra-test.datastax.com/.apps.astra-dev.datastax.com respectively, so we will have to get one other parameter for that.

Reply:
I have a strong preference for one single connection string rather than multiple environment variables. We can support both of course, and larger projects may prefer to go the multiple environment variable approach. But for a sample app it is much easier to copy one single string. Having several environment variables just adds more steps to getting the sample app up and running, and adds additional room for error.

That presumes that there's a place in Astra where you can copy the full connection string of course.

Reply:
There is one place, but the keyspace and the token needs to be appended to the URL manually by the user.